mlir/lib/Target/LLVMIR/AttrKindDetail.h

```diff
@@ -59,6 +59,19 @@ getAttrKindToNameMapping() {
   return kindNamePairs;
 }
 
+static llvm::DenseMap<llvm::StringRef, llvm::Attribute::AttrKind>
+getAttrNameToKindMapping() {
+  static auto attrNameToKindMapping = []() {
+    static llvm::DenseMap<llvm::StringRef, llvm::Attribute::AttrKind>
```

Comment:
```suggestion
   llvm::DenseMap<llvm::StringRef, llvm::Attribute::AttrKind>
```

No need for static in the initializer.